checker/src/main/java/org/checkerframework/checker/signedness/jdk.astub

```diff
@@ -3268,6 +3268,34 @@ class PrintStream {
 class RandomAccessFile {
     public void write(@PolySigned byte b[]);
     public void write(@PolySigned byte b[], int off, int len);
+    public void write(@PolySigned int b) throws IOException;
+    public final void writeByte(@PolySigned int v) throws IOException;
+    public final void writeShort(@PolySigned int v) throws IOException;
+    public final void writeChar(@PolySigned int v) throws IOException;
+    public final void writeInt(@PolySigned int v) throws IOException;
+    public final void writeLong(@PolySigned long v) throws IOException;
+    private native void writeBytes(@PolySigned byte b[], int off, int len) throws IOException;
+    public final void writeChars(String s) throws IOException {
+        int clen = s.length();
```

Comment:
Why does your stub file contain the implementation of `writeChars` but not of other methods?  I think it would be better to be consistent -- either the full file, or only method signatures for each annotated method.

Reply:
There is no need to implement other methods. Only method signatures for each annotated method is sufficient.
Only in ```writeChars``` method, the array was storing unsigned values, therefore, it was needed to be implemented.

Reply:
Can you tell me what goes wrong if you leave the method body out of this file?

Reply:
Nothing goes wrong if we use this method in a program. As signedness checker only runs on the method signature and not on its body. So I think it is not necessary to annotate this.

Reply:
Nothing goes wrong if we use this method in a program. As it works normally. So I think it is not necessary to annotate this.